.github/workflows/release.yml

```diff
@@ -45,3 +43,13 @@ jobs:
           RELEASER_CLIENT_SECRET: ${{ secrets.RELEASER_CLIENT_SECRET }}
           RELEASER_INSTALLATION_ID: ${{ secrets.RELEASER_INSTALLATION_ID }}
           DEPLOY_KEY: ${{ secrets.DEPLOY_KEY }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: 'https://npm.pkg.github.com'
+          node-version-file: '.nvmrc'
+      - name: Publish to GitHub Packages
+        run: npm run release:phase2
```

Comment:
I'm not sure whether this will work, since `release:phase2` also depends on `release:phase1` which bumps packages and `release:build` which rebuilds the package.

This *might* be fine because the bump script has this check at the beginning:
```json
const lastTag = await getLastTag(versionPrefix);
const commits = await getCommits(PATH, lastTag);
if (commits.length === 0 && !hasPackageJsonChanged(PATH)) {
  return;
}
```

`commits.length` should already be zero since we just did a release, therefore the new commit should be tagged, and `hasPackageJsonChanged` should return `false` since we're still in the same job as the publish, therefore our Git history should be on the new commit.

However, a safer approach may be to add a "phase5" which re-publishes. I opened a PR here that you can integrate into your PR: https://github.com/coveo/ui-kit/pull/3263